examples/03_datetime_encoder.py

```diff
@@ -280,8 +241,8 @@
 plt.show()
 
 # %%
-# We can see that the total seconds since Epoch and the hour of the day
-# are the most important feature, which seems reasonable.
+# We can see that the hour of the day, the temperature and the humidity
+# are the most important features, which seems reasonable.
 #
 # Conclusion
 # ----------
```

Comment:
Below :

In this example, we saw how to use the |DatetimeEncoder| to create features from a date column. 
-> In this example, we saw how to use the |DatetimeEncoder| to create features from a **datetime** column.